client/src/test/helper.ts

```diff
@@ -0,0 +1,48 @@
+import vscode from 'vscode';
+import path from 'path';
+
+export let doc: vscode.TextDocument;
+export let editor: vscode.TextEditor;
+export let documentEol: string;
+export let platformEol: string;
+
+/**
+ * Activates the vscode.lsp-sample extension
+ */
+export async function activate(docUri: vscode.Uri) {
+	// The extensionId is `publisher.name` from package.json
+	const ext = vscode.extensions.getExtension('Prisma.prisma-vscode')!;
+	await ext.activate();
+	try {
+		doc = await vscode.workspace.openTextDocument(docUri);
+		editor = await vscode.window.showTextDocument(doc);
+		await sleep(2000); // Wait for server activation
```

Comment:
Is there anything we can do to check the readiness of the server? Maybe like a ping command or something? This kind of arbitrary timeout often leads to test flakiness.